ext/compressors/zlib/zlib_compress.c

```diff
@@ -32,17 +32,15 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include <wiredtiger.h>
-#include <wiredtiger_ext.h>
-
 /*
  * We need to include the configuration file to detect whether this extension
- * is being built into the WiredTiger library.
+ * is being built into the WiredTiger library; application-loaded compression
+ * functions won't need it.
  */
-#include "wiredtiger_config.h"
-#ifdef _MSC_VER
-#define	inline __inline
-#endif
```

Comment:
This change is required unless we remove `inline` from `zlib_find_slot`.

I'm pretty sure lz4 has never been built on Windows, and snappy only uses it for big endian swapping.